core/tests/protractor_utils/forms.js

```diff
@@ -322,24 +322,38 @@ var UnicodeEditor = function(elem) {
 };
 
 var AutocompleteDropdownEditor = function(elem) {
-  var containerLocator = by.css('.select2-container');
-  var dropdownElement = element(by.css('.select2-dropdown'));
-  var searchInputLocator = by.css('.select2-search input');
+  var containerLocator = by.css('.e2e-test-exploration-category-dropdown');
+  var searchInputLocator = by.css(
+    '.mat-select-search-input.mat-input-element');
+  var categorySelectorChoice = by.css(
+    '.e2e-test-exploration-category-selector-choice');
+
   return {
     setValue: async function(text) {
-      await action.click('Container Element', elem.element(containerLocator));
+      await action.click(
+        'Container Element', elem.element(containerLocator), true);
       await action.waitForAutosave();
       // NOTE: the input field is top-level in the DOM, and is outside the
       // context of 'elem'. The 'select2-dropdown' id is assigned to the input
       // field when it is 'activated', i.e. when the dropdown is clicked.
       await action.sendKeys(
         'Dropdown Element',
-        dropdownElement.element(searchInputLocator),
-        text + '\n');
+        element(searchInputLocator),
+        text);
+
+      var searchInputLocatorText = element.all(by.cssContainingText(
+        '.e2e-test-exploration-category-selector-choice', text)).first();
+
+      await action.click(
+        'Dropdown Element Select',
+        searchInputLocatorText,
+        true);
```

Comment:
Why is this `true` passed only here and not in webdriverio?

Reply:
removed